src/JoomlaBrowser.php

```diff
@@ -1196,4 +1196,70 @@ public function createCategory($title, $extension = '')
 		$this->see('Category saved', '#system-message-container');
 		$this->checkForPhpNoticesOrWarnings();
 	}
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline()
+	{
+		$this->debug('I set the site to offline');
+		$this->doAdministratorLogin();
+		$this->amOnPage('/administrator/index.php?option=com_config');
+		$this->click(['id' => 'jform_offline1']);
+		$this->click("Save");
+		$this->see('Configuration saved', ['id' => 'system-message-container']);
+	}
+	/**
+	 * Site Online
+	 *
+	 * @param   \AcceptanceTester   $I  Acceptance Tester
+	 *
+	 * @return void
+	 */
+	public function setSiteOnline()
```

Comment:
Combine setSiteOffline and setSiteOnline into one function (with a parameter, defaulting to true)

```php
pubf setMaintenanceMode($enabled = true)
{
// ..
}
```